kmip/core/factories/secrets.py

```diff
@@ -48,14 +48,33 @@ def __init__(self):
                                                      '{2}')
 
     def create(self, secret_type, value=None):
+        """
+        Create a secret object of the specified type with the given value.
+
+        Args:
+            secret_type (ObjectType): An ObjectType enumeration specifying the
+                type of secret to create.
+            value (dict): A dictionary containing secret data. Optional,
+                defaults to None.
+
+        Returns:
+            secret: The newly constructed secret object.
+
+        Raises:
+            TypeError: If the provided secret type is unrecognized.
+
+        Example:
+            >>> factory.create(ObjectType.SYMMETRIC_KEY)
+            SymmetricKey(...)
```

Comment:
It just shows basic usage, where passing in the object type yields an empty object of that type. It could be more detailed but that would make the docstring really long. If it's confusing I can take it out.